libs/martech/attributes.js

```diff
@@ -11,6 +13,13 @@ export function processTrackingLabels(text, config, charLimit) {
   return analyticsValue;
 }
 
+function getHeaderCharLimit(str) {
+  const defaultLimit = 20;
+  if (str === null || str === undefined) return defaultLimit;
```

Comment:
Nit: I think this can be shorter
```suggestion
  if (!str) return defaultLimit;
```